site/gdocs/utils.tsx

```diff
@@ -68,10 +69,12 @@ export const useLinkedDocument = (
     } else if (!linkedDocument.published) {
         errorMessage = `Article with slug "${linkedDocument.slug}" isn't published.`
     }
+    const subdirectory =
```

Comment:
for my understanding, just confirming this is a side fix, not a consequence of something that happened somewhere else in this PR I might have missed?